### ISSUE TITLE
potion effect duration

### ISSUE THREAD

**Issue by Boilll** [SkriptLang/Skript#5987]
### Skript/Server Version

```markdown
Skript version: 2.7.0 (lastest)
Server version: Paper 1.20.1 124
```

### Bug Description

Applying potion effect only sets duration to 15 seconds.
In the previous version, it didn't have this bug.
```
apply swiftness 5 to player for 10 seconds                             -> gives swiftness 5 for 15 seconds
apply fire resistance 1 without particles to player for 100 seconds    -> gives fire resistance for 15 seconds
```
And all the potion effect command does same thing, giving effect for only 15 seconds.

### Expected Behavior

It should apply the seconds I set, but it gives only 15 seconds (not more, not less).

### Steps to Reproduce

I tested it with many durations, but it did not work.
But with skBee, the infinite potion effect works right.
```
apply infinite potion effect of night vision of tier 1 without particles to player
```

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by DelayedGaming** [comment]
Can confirm [these line](https://github.com/SkriptLang/Skript/blob/9c82fed023ff02626cc22233a1d1c78bb65a3d6c/src/main/java/ch/njol/skript/effects/EffPotion.java#L104-L105) breaks the pattern which includes duration parameter, causing [this line](https://github.com/SkriptLang/Skript/blob/9c82fed023ff02626cc22233a1d1c78bb65a3d6c/src/main/java/ch/njol/skript/effects/EffPotion.java#L123) to set the duration to 15 seconds regardless.

**Comment by MILOSZPOL** [comment]
I've this bug too

**Comment by MILOSZPOL** [comment]
This version is verry bugging old versions is better

**Comment by 4L0N50asd** [comment]
Still having this bug in version 2.7.0, confirm that
`apply infinite night vision without particles to player` gives an error, also tested with these and i got instead a 15s potion effect:
- `apply night vision without particles to the player for 30 days`
- `apply night vision without particles to the player for 1 day`
- `apply night vision without particles to the player for 1 hour`
- `apply night vision without particles to the player for 1 minute`

```
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! [Skript] Severe Error:
[00:51:36 ERROR]: #!#! Unexpected error while executing effect command 'apply infinite night vision without any particles to player' by 'DefKta'
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Something went horribly wrong with Skript.
[00:51:36 ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[00:51:36 ERROR]: #!#! You should report it at https://github.com/SkriptLang/Skript/issues. Please copy paste this report there (or use paste service).
[00:51:36 ERROR]: #!#! This ensures that your issue is noticed and will be fixed as soon as possible.
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Stack trace:
[00:51:36 ERROR]: #!#! java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.effects.EffPotion.init(EffPotion.java:105)
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:266)
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Effect.parse(Effect.java:76)
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.command.Commands.handleEffectCommand(Commands.java:212)
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.command.Commands$2$1.call(Commands.java:316)
[00:51:36 ERROR]: #!#!     at Skript.jar//ch.njol.skript.command.Commands$2$1.call(Commands.java:313)
[00:51:36 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftFuture.run(CraftFuture.java:88)
[00:51:36 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:480)
[00:51:36 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1480)
[00:51:36 ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:447)
[00:51:36 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1394)
[00:51:36 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1171)
[00:51:36 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318)
[00:51:36 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Version Information:
[00:51:36 ERROR]: #!#!   Skript: 2.7.0 (latest)
[00:51:36 ERROR]: #!#!     Flavor: skriptlang-github
[00:51:36 ERROR]: #!#!     Date: 08:55:03.744299
[00:51:36 ERROR]: #!#!   Bukkit: 1.20.1-R0.1-SNAPSHOT
[00:51:36 ERROR]: #!#!   Minecraft: 1.20.1
[00:51:36 ERROR]: #!#!   Java: 17.0.1 (Java HotSpot(TM) 64-Bit Server VM 17.0.1+12-LTS-39)
[00:51:36 ERROR]: #!#!   OS: Windows 10 amd64 10.0
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Server platform: Paper
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Current node: null
[00:51:36 ERROR]: #!#! Current item: null
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Thread: Server thread
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! Language: english
[00:51:36 ERROR]: #!#! Link parse mode: STRICT
[00:51:36 ERROR]: #!#!
[00:51:36 ERROR]: #!#! End of Error.
[00:51:36 ERROR]: #!#!
```

/skript info:
![image](https://github.com/SkriptLang/Skript/assets/67982577/71dca340-1815-438b-93ba-0981e23817da)

**Comment by sovdeeth** [comment]
> Still having this bug in version 2.7.0, confirm that `apply infinite night vision without particles to player` gives an error, also tested with these and i got instead a 15s potion effect:

Yes, it has been fixed and will be out in 2.7.1. See the "completed" tag.

**Comment by Boilll** [comment]
> > Still having this bug in version 2.7.0, confirm that `apply infinite night vision without particles to player` gives an error, also tested with these and i got instead a 15s potion effect:
> 
> Yes, it has been fixed and will be out in 2.7.1. See the "completed" tag.

That's a nice news
